tests/test_srv6.py

```diff
@@ -313,6 +320,448 @@ def test_srv6(self, dvs, testlog):
         assert nexthop_entries == get_exist_entries(self.adb.db_connection, "ASIC_STATE:SAI_OBJECT_TYPE_NEXT_HOP")
         assert route_entries == get_exist_entries(self.adb.db_connection, "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY")
 
+class TestSrv6Vpn(object):
+    def setup_db(self, dvs):
+        self.pdb = dvs.get_app_db()
+        self.adb = dvs.get_asic_db()
+        self.cdb = dvs.get_config_db()
+
+    def create_srv6_vpn_route(self, routeip, nexthop, segsrc, vpn_sid, ifname):
+        table = "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY"
```

Comment:
For SRv6 VPN feature, it need FRR BGPd to learn routes as SRv6 route which need two DUTs at least in topology. There is no vtysh command to configure static SRv6 route.
So we have to use APP_DB in swss UT.

Reply:
Thanks @shuaishang is it possible to validate fpmsyncd to APP_DB path by linux route commands?

Reply:
The SRv6 VPN route didn't install on Linux since the Linux doesn't support it yet... In our case, Zebra will filter out such kind of routes.
It's the same issue to test it on sonic-vs because Linux data plane can't forward SRv6 VPN route.